script/seed.ts

```diff
@@ -671,97 +671,97 @@ const seed = async () => {
-      [26, "Fusionner avec son double", 13, null, 22],
+      [24, "Fuir et laisser ton autre franchir la Brèche", 13, null, 20, null],
+      [25, "Combattre son double et prendre sa place", 13, 11, 21, 0],
+      [26, "Fusionner avec son double", 13, null, 22, 0],
 
-      [27, "Accepter de donner ta mémoire au Spectre", 15, null, 16],
-      [28, "Refuser et affronter les ombres", 15, null, 16],
+      [27, "Accepter de donner ta mémoire au Spectre", 15, null, 16, null],
+      [28, "Refuser et affronter les ombres", 15, null, 16, null],
 
-      [29, "Suivre la lumière dans les souterrains", 16, null, 8],
-      [30, "Rebrousser chemin", 16, null, 8],
+      [29, "Suivre la lumière dans les souterrains", 16, null, 8, null],
+      [30, "Rebrousser chemin", 16, null, 8, null],
 
-      [31, "Prier auprès des fanatiques", 17, null, 18],
-      [32, "Tenter de dérober un artefact", 17, null, 18],
+      [31, "Prier auprès des fanatiques", 17, null, 18, null],
+      [32, "Tenter de dérober un artefact", 17, null, 18, null],
 
-      [33, "Purifier ta Marque à l’autel", 18, null, 8],
-      [34, "Corrompre davantage ta Marque à l’autel", 18, null, 8],
-      [35, "Reprendre la route vers la Tour du Voile", 16, null, 7],
-      [36, "Quitter les Temples et rejoindre la Tour du Voile", 18, null, 7],
+      [33, "Purifier ta Marque à l’autel", 18, null, 8, null],
+      [34, "Corrompre davantage ta Marque à l’autel", 18, null, 8, null],
+      [35, "Reprendre la route vers la Tour du Voile", 16, null, 7, null],
+      [36, "Quitter les Temples et rejoindre la Tour du Voile", 18, null, 7, null],
       [
         37,
         "Se rendre aux Ruines d’Ashtorath pour réveiller la Reine Écorchée",
         7,
         null,
-        15,
+        15,  null
       ],
       [
         38,
         "Se rendre aux Temples pour éveiller le Veilleur Aveugle",
         7,
         null,
-        17,
+        17, null
       ],
       [
         39,
         "Se rendre sous le Lac Noir pour éveiller la Mère Cendre",
         7,
         null,
-        19,
+        19, null
       ],
       [
         40,
         "Éveiller la Mère Cendre en offrant un fragment de toi-même",
         19,
         null,
-        8,
+        8, null
       ],
-      [41, "Refuser de troubler son sommeil et fuir discrètement", 19, null, 8],
-      [42, "Fin : Le Héros Oublié", 20, null, 23],
-      [43, "Fin : Le Brisé", 21, null, 23],
-      [44, "Fin : Le Souverain Gris", 22, null, 23],
-      [45, "revenir à l'accueil.", 23, null, 0],
+      [41, "Refuser de troubler son sommeil et fuir discrètement", 19, null, 8, null],
+      [42, "Fin : Le Héros Oublié", 20, null, 23, null],
+      [43, "Fin : Le Brisé", 21, null, 23, null],
+      [44, "Fin : Le Souverain Gris", 22, null, 23, null],
+      [45, "revenir à l'accueil.", 23, null, 0, null],
     ];
     for (const choice of choices) {
       await connection.execute(
```

Comment:
add take_or_give values as well (0 for take and 1 for give ? since it will be a boolean